### DISCUSSION TITLE
Reopening closed issues in mobile is really unintuitive

### DISCUSSION THREAD

**Discussion by lucacasonato** [community/community#9517] [Mobile] [answered]
If I get deep linked to a comment on a closed issue, it is not immediately obvious how to reopen the issue. There is no reopen button at the bottom. There is also no three dots visible that I can use to get more details. Instead I need to scroll all the way to the top of the issue, and then make sure to click on the right three dots (the ones next to the issue title, not the issue description), and only then do I get a "reopen issue" button.

Shouldn't the triple dot menu for acting on the issue itself be always visible (even if you are looking at comment #10)?

I am using the iOS app version 1.44.0.

**Comment by brianlovin** [comment, marked as answer]
We are going to move this `···` action to the top navigation bar so that it will be accessible from any point in the view. Thanks for taking the time to report this, and keep an eye out for the next update or two where this fix should be live.
